test/setup/main.tf

```diff
@@ -16,7 +16,7 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 15.0"
+  version = "~> 17.0"
```

Comment:
why is this needed?

Reply:
This change was present in one of the open PRs, hence I replicated it here. It isn't really needed for this PR specifically.

Reply:
Done.